docs/architecture/accounts.md

```diff
@@ -11,11 +11,11 @@ Tezos uses these types of accounts:
 These accounts have addresses that start with "tz1", "tz2", "tz3" or "tz4."
 Any wallet application or the Octez command-line tool can create implicit accounts.
 
-- Smart contract accounts (also known as _originated accounts_) store immutable code, mutable storage, tez (ꜩ), and tickets.
+- Smart contract accounts (a type of _originated account_) store immutable code, mutable storage, tez (ꜩ), and tickets.
```

Comment:
this implies that there are other types of originated accounts, which I don't think is true?

Reply:
Are Smart Rollup accounts not originated accounts?

Reply:
As told in a different thread, I don't think they are. If we want to reclassify them as accounts, we have to discuss this in a documentation venue (e.g. EffortDoc).

Reply:
I'm removing changes to this file from this PR and I'm working on clarifying accounts and addresses in this pr: https://github.com/trilitech/tezos-developer-docs/pull/252